source/tutorials/kubernetes.rst

```diff
@@ -3,170 +3,340 @@
-to suit your needs.
+* `What is Kubernetes?`_
+* `What is CRI+O?`_
+* `What is Kata Containers?`_
 
 Prerequisites
 *************
 
 This tutorial assumes you have already installed |CL|. For detailed
-instructions on installing |CL| on a bare metal system, follow the
-:ref:`bare metal installation tutorial<bare-metal-install-desktop>`.
-Learn about the benefits of having an up-to-date system for cloud
-orchestration on the :ref:`swupd-guide` page.
+instructions on installing |CL| on a bare metal system, follow the :ref:`bare
+metal installation tutorial<bare-metal-install-desktop>`.
 
-Before you install any new packages, update |CL| with the following command:
+Before you continue, update your |CL| installation with the following command:
 
 .. code-block:: bash
 
    sudo swupd update
 
-Install Kubernetes and CRI runtimes
-***********************************
+Learn about the benefits of having an up-to-date system for cloud
+orchestration on the :ref:`swupd-guide` page.
+
+
+Install Kubernetes
+******************
 
 Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
-runtimes, and networking plugins, are included in the `cloud-native-basic`_
-bundle.
+runtimes and :abbr:`CNI (Container Network Interface)` are included in the
+`cloud-native-basic`_ bundle.
 
-To install this framework, enter the following command:
+#. Install the cloud-native-basic bundle to get these components:
 
 .. code-block:: bash
 
    sudo swupd bundle-add cloud-native-basic
 
-.. note::
 
-   For more on networking plugins, see `Install pod network add-on`_.
+Configure host system
+=====================
 
-Configure Kubernetes
-********************
+This tutorial uses the basic default Kubernetes configuration for to get
+started. You can customize your Kubernetes configuration according to your
+specific deployment and security needs.
 
-This tutorial uses the basic default Kubernetes configuration for simplicity.
-You must define your Kubernetes configuration according to your specific
-deployment and your security needs.
+The Kubernetes administration tool, :command:`kubeadm`, performs some
+"`preflight checks`_" when initializing and starting a cluster. The steps
+below are necessary to ensure those `preflight checks`_ pass successfully.
 
-#. Enable IP forwarding to avoid kubeadm `preflight check`_ errors:
 
-   Create (or edit if it exists) the file :file:`/etc/sysctl.d/60-k8s.conf`
-   and include the following line:
+#. Enable IP forwarding:
 
-   .. code-block:: bash
+   - Create the file :file:`/etc/sysctl.d/60-k8s.conf` to set the
+     :command:`net.ipv4.ip_forward` parameter
+
+     .. code-block:: bash
+
+        sudo mkdir -p /etc/sysctl.d/
+
+        sudo tee /etc/sysctl.d/99-kubernetes-cri.conf > /dev/null <<EOF
+        net.bridge.bridge-nf-call-iptables  = 1
+        net.ipv4.ip_forward                 = 1
+        net.bridge.bridge-nf-call-ip6tables = 1
```

Comment:
Is this new guidance?  Previously only `net.ipv4.ip_forward = 1` is required.   Just searching through the web for these settings I found a forum where they also recommended  `net.bridge.bridge-nf-call-arptables = 1`

Reply:
Yes, according to https://kubernetes.io/docs/setup/production-environment/container-runtimes/ and the cloud-native-scripts.

Reply:
OK then.  What about arptables?  I saw that in some forum threads in addition to the above.  I think the above are when running kubernetes in a virtual machine already but am not 100% sure.